MkChrootTree.sh

```diff
@@ -70,6 +71,7 @@ function UsageMsg {
       printf '\t%-4s%s\n' '-m' 'Where to mount chroot-dev (default: "/mnt/ec2-root")'
       printf '\t%-4s%s\n' '-p' 'Comma-delimited string of colon-delimited partition-specs'
       printf '\t%-6s%s\n' '' 'Default layout:'
+      # shellcheck disable=SC2048
       for PART in ${DEFGEOMARR[*]}
```

Comment:
Is there any reason not to use this, to avoid needing to disable the shellcheck finding?

```suggestion
      for PART in "${DEFGEOMARR[@]}"
```

Reply:
Last time that section of code was updated was:

~~~
$ git log -S 'for PART in ${DEFGEOMARR[*]}'
commit c730ce590f97a4a3925a22a25c021ed24cb8fd06
Author: Thomas H Jones II <github@xanthia.com>
Date:   Mon Apr 6 16:34:56 2020 -0400

    Start creating chroot-mounter script
~~~

So, when I first saw the error, I changed to:

~~~
for PART in "${DEFGEOMARR[*]}"
~~~

which resulted in:
~~~
In MkChrootTree.sh line 74:
      for PART in "${DEFGEOMARR[*]}"
                  ^----------------^ SC2066 (error): Since you double quoted this, it will not word split, and the loop will only run once.

For more information:
  https://www.shellcheck.net/wiki/SC2066 -- Since you double quoted this, it ...
~~~

And was in a rush to get the test-AMI built, so just excepted the section and never came back to it. 😛 

I'll go make the `*` to `@`, now.